src/services/firebase.js

```diff
@@ -22,99 +21,37 @@ if (!firebase.apps.length) {
     firebase.app(); // if already initialized, use that one
 }
 
-
-const uiConfig = {
-    callbacks: {
-        signInSuccessWithAuthResult: function(authResult, redirectUrl) {
-            // User successfully signed in.
-            // Return type determines whether we continue the redirect automatically
-            // or whether we leave that to developer to handle.
-            console.log(authResult)
-
-            if (authResult.additionalUserInfo.isNewUser) {
-                // write user to db
-                createNewUser(
-                    authResult.user.uid,
-                    authResult.user.displayName,
-                    authResult.user.photoURL,
-                    authResult.user.email,
-                    authResult.user.phoneNumber
-                );
-                window.location.assign('/test')
-            } else {
-                window.location.assign('/');
-            }
-            // Manually redirect to EditProfile if new user, otherwise home
-        
-            return false;
-        },
-        uiShown: function() {
-            // The widget is rendered.
-            // Hide the loader.
-            // document.getElementById('loader').style.display = 'none';
-            console.log('uiShown callback func')
-        }
-    },
-    // Will use popup for IDP Providers sign-in flow instead of the default, redirect.
-    signInFlow: 'popup',
-    signInSuccessUrl: '/',
-    signInOptions: [
-        // Leave the lines as is for the providers you want to offer your users.
-        firebase.auth.GoogleAuthProvider.PROVIDER_ID,
-        // firebase.auth.FacebookAuthProvider.PROVIDER_ID,
-        // firebase.auth.TwitterAuthProvider.PROVIDER_ID,
-        // firebase.auth.GithubAuthProvider.PROVIDER_ID,
-        firebase.auth.EmailAuthProvider.PROVIDER_ID,
-        // firebase.auth.PhoneAuthProvider.PROVIDER_ID
-    ],
-    // Terms of service url.
-    tosUrl: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO',
-    // Privacy policy url.
-    privacyPolicyUrl: 'https://www.youtube.com/watch?v=y6120QOlsfU&ab_channel=Darude'
+export const auth = firebase.auth;
+export const db = firebase.database();
+export const createNewUser = async (uid, name, photo, email, phone) => {

```

Comment:
Perhaps change to take in an options object with key-value pairs instead of multiple parameters?

Reply:
Done